Frontend/src/services/axios.tsx

```diff
@@ -0,0 +1,105 @@
+'use client';
+import axios from "axios";
+import { ACCESS_TOKEN, REFRESH_TOKEN } from "@/hooks/useAccessToken";
+
+const getAccessToken = () => {
+    if (typeof window === "undefined") {
+        return null;
+    }
+    return localStorage.getItem(ACCESS_TOKEN);
+}
+
+export const authApi = axios.create({
+  headers: {
+      "Content-Type": "application/json",
+      "Accept": "application/json",
+      "Authorization": "Bearer " + getAccessToken(),
+  }
+})
+
+const api = axios.create({
+    headers: {
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+    },
+});
+
+api.interceptors.request.use(
+    (config) => {
+        const tokenStr = sessionStorage.getItem("AUTH");
+        try {
+            if (tokenStr) {
+                const token = JSON.parse(tokenStr);
+                if (token) {
+                    config.headers[
+                        "Authorization"
+                    ] = `Bearer ${token?.access_token}`;
+                }
+            }
+        } catch (e) {}
```

Comment:
Why are we not logging the error message?

Reply:
logged the error

Reply:
Ok